src/main.cc

```diff
@@ -610,7 +615,7 @@ int run_server(StringView session, StringView init_command,
     return 0;
 }
 
-int run_filter(StringView keystr, StringView commands, ConstArrayView<StringView> files, bool quiet)
+int run_filter(StringView keystr, StringView commands, ConstArrayView<StringView> files, bool quiet, bool readonly)
```

Comment:
That can be arranged, I modified this function for the sake of consistence with the `run_server` function, since they both declared options.